test-result-summary-client/src/utils/perf.js

```diff
@@ -1,21 +1,17 @@
 
-export const getParserProps = async () => {
-    let parserProps = await fetch( `/api/getParserProps`, {
+export const getParserProps = async (benchmarkName) => {
+    let parserProps = await fetch( `/api/getParserProps?benchmarkName=${benchmarkName}`, {
             method: 'get'
         }
     );
     parserProps = await parserProps.json();
     return parserProps;  
 }
 
-export function getMetricProps(parserProps,benchmarkName,curMetric) {
-    let benchmarkMetric = parserProps['benchmarkMetric'];
+export function getMetricProps(parserProps,curMetric) {
```

Comment:
- We do not need another layer. Please remove getMetricProps (). `getParserProps() ` should return the result.
- IMO, there is no need to have this file as we only have one function in this file and it does nothing but calling the API. We can just call the API whenever it is needed. However, @sophiaxu0424 I will leave it to you to decide.

Reply:
Will remain using the `perf.js` file here, in case we need to add other BenchmarkMetric info inside or more info would be added to BenchmarkMetric.js in future.